### DISCUSSION TITLE
Release 4.4.1?

### DISCUSSION THREAD

**Discussion by NeilSorensen** [remotemobprogramming/mob#377] [Q&A] [answered]
The changelog at https://mob.sh/CHANGELOG.html shows a 4.4.1 release, but I don't see it in the list of releases here on Github.

Am I missing something?

**Comment by hollesse** [comment]
Hey @NeilSorensen I just forgot to release the new version. I will release it as soon as possible. But I am on a conference right now.

**Comment by hollesse** [comment, marked as answer]
@NeilSorensen release is done!
